src/mjolnir/ingest_transit.cc

```diff
@@ -340,9 +349,12 @@ write_stops(Transit& tile, const tile_transit_info_t& tile_info, feed_cache_t& f
                     NodeType::kMultiUseTransitPlatform, false, prev_id);
       }
     }
-    // We require platforms as well so if we didnt add at least 1 we need to fake one now
+    // This really shouldn't happen, as platform IDs are referenced by the
+    // stop_times.txt and the whole downstream logic would get messed up
+    // can only happen if there was an entrance and a platform at the same station
+    // without that station being referenced by their parent_station atttributes
     if (tile.nodes_size() == node_count) {
-      LOG_WARN("Generated platform for station " + station_as_stop->stop_id);
+      LOG_ERROR("Generated platform for station " + station_as_stop->stop_id);
```

Comment:
we should put a TODO  here that peels back the changes above as well so that we can completely skip this stop. now it will also be possible that other things refer to this in which case we will have more errors.. but as best we can we should try to be robust to it. maybe after more imports it will be obvious the best way to recover from such a screwed up feed